logstash-core-event-java/src/main/java/org/logstash/Event.java

```diff
@@ -76,9 +76,19 @@ public Event(Map data)
         this.metadata_accessors = new Accessors(this.metadata);
 
         this.cancelled = false;
-        this.timestamp = initTimestamp(data.get(TIMESTAMP));
+
+        Object provided_timestamp = data.get(TIMESTAMP);
```

Comment:
Yeah we should be using unicode more anyway, like `Object providedⓉimestamp ...` ;P

+1 on more java-ish style camelCase.